baseline_models/models/aquatics_climatology.R

```diff
@@ -1,50 +1,30 @@
 #'# Ecological Forecasting Initiative Null Model
 
 #'## Set-up
-
 print(paste0("Running Creating Daily Terrestrial Forecasts at ", Sys.time()))
 
-#'Load renv.lock file that includes the versions of all the packages used
-#'You can generate using the command renv::snapshot()
-
 #' Required packages.
 #' EFIstandards is at remotes::install_github("eco4cast/EFIstandards")
 library(tidyverse)
 library(lubridate)
 library(aws.s3)
 library(jsonlite)
 library(imputeTS)
-
-
 #' set the random number for reproducible MCMC runs
 set.seed(329)
 
-#'Generate plot to visualized forecast
-generate_plots <- TRUE
-#'Is the forecast run on the Ecological Forecasting Initiative Server?
-#'Setting to TRUE published the forecast on the server.
-efi_server <- TRUE
-
-#' List of team members. Used in the generation of the metadata
-#team_list <- list(list(individualName = list(givenName = "Quinn", surName = "Thomas"),
-#                       id = "https://orcid.org/0000-0003-1282-7825"),
-#                  list(individualName = list(givenName = "Others",  surName ="Pending")),
-#)
+config <- yaml::read_yaml("challenge_configuration.yaml")
 
 #'Team name code
 team_name <- "climatology"
```

Comment:
@rqthomas  do I need to register this model (and the persistanceRW) for `usgsrc4cast` challenge before submitting?

Reply:
the `neon4cast::submit()` will work for `usgsrc4cast` project_id since they both are going to the same submission bucket.

Reply:
Yes - a model_id is unique to a project_id so it needs to be registered.